packages/pwa-buildpack/lib/WebpackTools/plugins/__tests__/ServiceWorkerPlugin.spec.js

```diff
@@ -137,14 +164,18 @@ test('.apply uses `InjectManifest` when `injectManifest` is `true`', () => {
         injectManifestConfig
     });
 
-    const fakeCompiler = {};
+    const fakeTap = jest.fn();
+    const fakeCompiler = { hooks: { afterEmit: { tap: fakeTap } } };
```

Comment:
Since we repeat these lines frequently let's pull them out to the top of the file and re-use them. Don't forget to clear the `fakeTap` mock between tests.

Reply:
We don't need to since jest will auto clear function mocks after each test.